data/event-schemas/schema/m.room.topic.yaml

```diff
@@ -1,7 +1,14 @@
 ---
 allOf:
   - $ref: core-event-schema/state_event.yaml
-description: 'A topic is a short message detailing what is currently being discussed in the room.  It can also be used as a way to display extra information about the room, which may not be suitable for the room name. The room topic can also be set when creating a room using `/createRoom` with the `topic` key.'
+description: |- 
+    A topic is a short message detailing what is currently being discussed in the room.
+    It can also be used as a way to display extra information about the room, which may not
+    be suitable for the room name.
+    The room topic can also be set when creating a room using `/createRoom` with the `topic` key.'
+    
+    If a room has an `m.room.topic` event with an absent, null, or empty `topic`
```

Comment:
So it should be noted, again as with `m.room.name`, that [Synapse doesn't allow a topic](https://github.com/element-hq/synapse/blob/ebad618bf0f4a7cd8adb5c65d6025d320387b492/synapse/events/validator.py#L213-L215) without a string topic key.

So this present behavior doesn't exist there, but I need to survey the other homeservers.